doc/stdlib/io-network.rst

```diff
@@ -338,41 +318,35 @@ General I/O
 
    Returns the file descriptor backing the stream or file. Note that this function only applies to synchronous ``File``\ 's and ``IOStream``\ 's not to any of the asynchronous streams.
 
-.. function:: redirect_stdout()
+.. function:: redirect_stdout([stream]) -> (rd, wr)
 
    .. Docstring generated from Julia source
 
-   Create a pipe to which all C and Julia level ``STDOUT`` output will be redirected. Returns a tuple ``(rd,wr)`` representing the pipe ends. Data written to ``STDOUT`` may now be read from the rd end of the pipe. The wr end is given for convenience in case the old ``STDOUT`` object was cached by the user and needs to be replaced elsewhere.
-
-.. function:: redirect_stdout(stream)
-
-   .. Docstring generated from Julia source
-
-   Replace ``STDOUT`` by stream for all C and Julia level output to ``STDOUT``\ . Note that ``stream`` must be a TTY, a ``Pipe`` or a ``TCPSocket``\ .
+   Create a pipe to which all C and Julia level :obj:`STDOUT` output will be redirected. Returns a tuple ``(rd, wr)`` representing the pipe ends. Data written to :obj:`STDOUT` may now be read from the ``rd`` end of the pipe. The ``wr`` end is given for convenience in case the old :obj:`STDOUT` object was cached by the user and needs to be replaced elsewhere.
```

Comment:
is the "``stream`` [if given] must be a TTY, a ``Pipe`` or a ``TCPSocket``" restriction from the deleted docstring accurate?